jobs/uaa-customized/spec

```diff
@@ -9,7 +9,23 @@ templates:
   bin/health_check: bin/health_check
 
   resources/oss/stylesheets/application.css: resources/oss/stylesheets/application.css
+  resources/oss/stylesheets/govuk-frontend-2.11.0.min.css: resources/oss/stylesheets/govuk-frontend-2.11.0.min.css
   resources/oss/images/product-logo.png: resources/oss/images/product-logo.png
   resources/oss/images/square-logo.png: resources/oss/images/square-logo.png
 
-properties: {}
+  assets/fonts/bold-a2452cb66f-v1.woff2: assets/fonts/bold-a2452cb66f-v1.woff2
```

Comment:
We may wish to change the css file to point to `/resources/fonts` and put these there, so UAA knows what to do with them